### DISCUSSION TITLE
WebHook Verification?

### DISCUSSION THREAD

**Discussion by daniel89fg** [netflie/whatsapp-cloud-api#57] [Q&A] [answered]
Hello.

Does this library have verification to connect the Facebook WebHook? If so, how would it be?

Thanks.

**Comment by aalbarca** [comment, marked as answer]
This library doesn't support webhooks.

**Reply by daniel89fg** [reply]
Is it planned to add support?

**Reply by aalbarca** [reply]
Some news here: https://github.com/netflie/whatsapp-cloud-api/discussions/65

**Reply by aalbarca** [reply]
@daniel89fg 2.x version has been released! https://github.com/netflie/whatsapp-cloud-api/discussions/75

**Comment by ithebesha** [comment]
Can You Help Me To Run The Project
